### PR TITLE
[13.0][MIG] stock_split_picking: Migration to 13.0

### PR DESCRIPTION
**Pull request by xavierpiernas**
Standard migration

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍 . Maybe you can add yourself to contributors, if you want.

**Review by Hafdal-la — APPROVED** [review]

File: stock_split_picking/README.rst
Comment (CarlosRoca13) [inline stock_split_picking/README.rst:17]:
```suggestion
    :target: https://github.com/OCA/stock-logistics-workflow/tree/13.0/stock_split_picking
```
You have to generate the readme for the 13.0 version

File: stock_split_picking/readme/CONTRIBUTORS.rst
Comment (CarlosRoca13) [inline stock_split_picking/readme/CONTRIBUTORS.rst:6]:
```suggestion
* Andrius Preimantas <andrius@versada.eu>
* Your name and surname <youremail if u want>
```
I think you would like to be a contributor

**Review by CarlosRoca13 — COMMENTED** [review]
It's important to generate the readme following the instructions that we find on the readme of https://github.com/OCA/maintainer-tools . 

I hope I have been helpful

**Comment by rousseldenis** [comment]
> It's important to generate the readme following the instructions that we find on the readme of https://github.com/OCA/maintainer-tools .
> 
> I hope I have been helpful

As readme fragments folder has been populated, the README should be generated automatically with oca-bot after merge

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by CarlosRoca13 — APPROVED** [review]
@rousseldenis Okey, thanks for the info

So I aprove too :+1:

**Pull request merged**
